package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A GitHub action to open, approve and merge a PR of changes generated during a release on a protected branch",
   "main": "dist/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --detectOpenHandles",
```

Comment:
ooh what is `detectOpenHandles`?

Reply:
Oh yeah, I forgot about that. I got a warning about failing to quit in time and one of the things it said was to try adding this option so you could see if there was something that wasn't being closed/shutdown/terminated correctly (I don't remember the exact wording) so I tried it. I haven't had the warning any more, not sure if that's related or a coincidence.